### DISCUSSION TITLE
Store the skin used when recording a replay

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#23736] [Ideas]
Record the skin used when completing the grade. If have the same skin as the record during playback, use that skin for playback.

This can restore the feeling of defeating Beatmap at that time.

**Comment by monochrome22** [comment]
It was already asked, the answer is here: https://github.com/ppy/osu/discussions/21729#discussioncomment-4455974.
But this discussion thread will be used for tracking this specific feature (decided by peppy).
